wafer/talks/models.py

```diff
@@ -129,7 +129,8 @@ class Meta:
 
     notes = models.TextField(
         null=True, blank=True,
-        help_text=_("Any notes for the conference organisers?"))
+        help_text=_("Any notes for the conference organisers?  "
+                    "Attendees cannot read those."))
```

Comment:
The "those" seems a bit distant. How about: These are not visible to attendees or the public.